task/04-arrays-tasks.js

```diff
@@ -337,9 +357,9 @@ function sortDigitNamesByNumericOrder(arr) {
  *   [ 1, 10, 100, 1000 ]  => 1111
  */
 function getItemsSum(arr) {
-   throw new Error('Not implemented');
+   return (arr.length) ? arr.reduce((sum, currentItem) => sum += currentItem) : 0;
```

Comment:
redundant logic, we can use just reduce with initial value 0
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce